website/docs/wallet/slashing-protection.md

```diff
@@ -18,6 +18,12 @@ To protect itself from accidentally being slashed due to some software bug or ot
 
 **This database is kept, by default, within the validator's wallet directory in a file called validator.db**. However, instead of manually copying and dealing with this file, Prysm provides tools to export and import your slashing protection history into a standard format supported by all eth2 clients, such as Lighthouse, Teku, and Nimbus. This standard format is useful as well if you want to move your validator between computers securely. You can export your slashing protection history from one computer as a file and import it safely into the other.
 
+## How to use slashing protection
+
+
+Basic slashing protection is **enabled** by default using a database that keeps track of objects your validator has previously signed in order to prevent it from signing the same message again causing a violation and getting slashed. If you want to use a more advanced, *remote* slashing protection, see our section on how to use **slasher** [here](/docs/prysm-usage/slasher).
```

Comment:
```suggestion
Basic slashing protection is **enabled** by default using a database that keeps track of objects your validator has previously signed in order to prevent it from signing the same message again, causing a violation and getting slashed. If you want to use a more advanced, *remote* slashing protection, see our section on how to use **slasher** [here](/docs/prysm-usage/slasher).
```
I am not sure whether a comma is required here, but I think the sentence reads better this way. Otherwise it's very long and doesn't have any punctuation.